composer.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "openeuropa/oe_content",
+    "description": "OpenEuropa Drupal module template.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": "^7.1",
+        "drupal/core": "~8.6",
+        "drupal/rdf_entity": "^1.0.0-alpha7",
+        "easyrdf/easyrdf": "0.10.0-alpha1 as 0.9.2"
+    },
+    "require-dev": {
+        "composer/installers": "^1.2",
+        "drupal/config_devel": "~1.2",
+        "drupal/drupal-extension": "^4.0.0@alpha",
+        "drupal-composer/drupal-scaffold": "^2.2",
+        "drush/drush": "^9",
+        "openeuropa/code-review": "~0.2",
+        "openeuropa/task-runner": "^0.7",
+        "webflo/drupal-core-require-dev": "~8.6"
+    },
+    "suggest": {
+        "drupal/console": "^1"
+    },
+    "scripts": {
+        "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+        "post-install-cmd": "./vendor/bin/run drupal:site-setup",
+        "post-update-cmd": "./vendor/bin/run drupal:site-setup"
+    },
+    "repositories": {
+        "drupal": {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    },
+    "extra": {
```

Comment:
Add a patches extra for the recursion fix:

```
  "composer-exit-on-patch-failure": true,
        "patches": {
            "drupal/core": {
                "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-07-05/2943172-kernel-test-base-3.patch"
            }
        },
```

And make sure you also require-dev above:

```
"cweagans/composer-patches": "~1.0",
```